python/cuml/cuml/manifold/umap.pyx

```diff
@@ -845,6 +849,35 @@ class UMAP(UniversalBase,
         del X_m
         return embedding
 
+    def __getstate__(self):
```

Comment:
This works for the pickling test, but not for regular GPU training followed by CPU inference. I think that the code should be put into lazily-computed properties instead.